graylog2-web-interface/package.json

```diff
@@ -90,9 +78,8 @@
     "less-loader": "^2.2.1",
     "mocha": "^2.3.4",
     "phantomjs-prebuilt": ">=1.9",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "^3.0.0-beta.3",
```

Comment:
Sounds good to me, also seeing the rhythm of beta releases we will most likely use a stable version by the end of our release cycle 👍